tfhe/src/shortint/server_key/mod.rs

```diff
@@ -1685,3 +1496,65 @@ impl ServerKey {
             .min_by_key(|op| op.number_of_pbs())
     }
 }
+
+pub(crate) fn apply_blind_rotate(
+    bootstrapping_key: &ShortintBootstrappingKey,
+    in_buffer: &LweCiphertextView<u64>,
+    acc: &mut GlweCiphertextMutView<u64>,
```

Comment:
maybe make generic for the buffer containers ?

Reply:
Done